menu.py

```diff
@@ -756,6 +778,7 @@ def run(self):
 
             menu += [
                 ('lp', 'Load preset gamma curve', self.preset_gamma_menu_select),
+                ('ig', 'Import gamma curve from VCGT file', self.import_vcgt),
```

Comment:
move this to the detailed menu?

Reply:
Is that the "Adjust gamma curve" menu?

Fine by me, although normally all that would be needed in my situation is a "ig <filename>" followed by "Pw" to write the values to the projector.  I do find it useful though to use the plot window to verify the correct table has been loaded.  Speaking of that: I noticed that if I open the plot window first, then load a VCGT file it will update the plot window just fine.  However, if I load the VCGT first, then try to open the plot window it results in an error ('<' not supported between instances of 'NoneType' and 'int' / error occured print stack trace?).  My Python skills are still very limited (first exposure to it for this project), so I don't know what to do about that.  Any ideas?

Reply:
Yes. I would put it right below `('Pr', 'Read raw table from projector', lambda _: self.gamma.read()),` as this has the most similar end result.

Regarding the crash I would need to see the actual stack trace to know why it crashed, but it might be related to the existing raw gamma cure support. You can test this by using 'Pr' instead of your command. It that also crashes the bug is yours.

Reply:
Moved the menu entry.

The stack trace:

```
Traceback (most recent call last):
  File "F:\Programming\GitHub\jvcprojectortools\menu.py", line 194, in run
    self.menu.run()
  File "F:\Programming\GitHub\jvcprojectortools\menu.py", line 769, in run
    self.run_autoplot(gammahist)
  File "F:\Programming\GitHub\jvcprojectortools\menu.py", line 822, in run_autoplot
    self.clear_plot_draw_grid()
  File "F:\Programming\GitHub\jvcprojectortools\menu.py", line 647, in clear_plot_draw_grid
    vlines.sort()
TypeError: '<' not supported between instances of 'NoneType' and 'int'
```

Reply:
That crash is because the raw gamma curves don't have hard and soft clip values:
`(self.gamma.ihardclip, 'Hard Clip', 3),
 (self.gamma.isoftclip, 'Soft Clip', 4),`

It is not directly related to your change.